### ISSUE TITLE
Base resource type not found (Git)

### ISSUE THREAD

**Issue by rodolfocugler** [neumayer/concourse-arm-worker#5]
followed this [tutorial](https://martin-grigorov.medium.com/concourseci-on-arm64-229833883ca9) to start a Concourse Worker in my Raspberry Pi 4 (8Gb RAM).

I was able to spin up the docker container and "bind" to my web instance.

| name | containers | platform | tags | team | state | version | age | garden address | baggageclaim URL | active tasks | resource types|
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 2eb2807e9b4c | 1 | linux | none | none | running | 2.2 | 51m41s | 127.0.0.1:45427 | http://127.0.0.1:35189 | 0 | registry-image |

Concourse Web checks successfully the resources with registry-image as type, but I get an error when trying to check resources like git as type.

`base resource type not found: git`

Obs.: the OS is `Linux raspberrypi 5.8.0-1015-raspi #18-Ubuntu SMP PREEMPT Fri Feb 5 06:09:58 UTC 2021 aarch64 aarch64 aarch64 GNU/Linux`

**Comment by neumayer** [comment]
We only register the registry-image resource. So this is as intended, but yeah, it requires to have at least one amd64 worker in addition (with a standard release of concourse that contains all the standard resource types).

**Comment by rodolfocugler** [comment]
Got it. Thanks 

Only to let you know and in case of someone else has the same issue:

I spent some time investigating how to use git and time resource types with only one arm64 worker instance. Then I could run those resources when I copied the resource-types from the concourse release (v7.0.0) folder to my concourse folder inside the container.

Example:
`-v $(pwd)/concourse/resource-types/git:/usr/local/concourse/resource-types/git`

I copied all resources except registry-image. However, I tested only git, time and docker-resource.
